src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/AwsDatasetModification.kt

```diff
@@ -74,6 +80,21 @@ class AwsDatasetModification private constructor(
         logger.info("AWS resources released")
     }
 
+    private fun cleanUp(infrastructure: Infrastructure<*>) {
+        val jiraHome = infrastructure.jira.jiraHome
+        val dirsToPurge = listOf(
```

Comment:
should we purge import and export directories as well just in case? @wyrzyk what do you think?

Reply:
👍